pandas/core/indexes/base.py

```diff
@@ -5290,6 +5291,27 @@ def _add_logical_methods_disabled(cls):
 Index._add_comparison_methods()
 
 
+class IndexGroupbyGroups(dict):
+    def __repr__(self):
```

Comment:
this just need to call pandas.io.formats.printing.pprint_thing (with the option passed)